docs/resources/instancev2.md

```diff
@@ -392,11 +394,9 @@ resource "gcore_instancev2" "instance" {
 - `configuration` (Block List) Parameters for the application template from the marketplace (see [below for nested schema](#nestedblock--configuration))
 - `keypair_name` (String) Name of the keypair to use for the instance
 - `last_updated` (String)
-- `metadata` (Block List, Deprecated) (see [below for nested schema](#nestedblock--metadata))
 - `metadata_map` (Map of String) Create one or more metadata items for the instance
 - `name` (String) Name of the instance.
 - `name_template` (String) Instance name template. You can use forms 'ip_octets', 'two_ip_octets', 'one_ip_octet'
-- `name_templates` (List of String, Deprecated) List of instance names which will be changed by template. You can use forms 'ip_octets', 'two_ip_octets', 'one_ip_octet'
```

Comment:
we have name_templates field for instance creation

Reply:
or is it naming for terraform? because I see that metadata_map is used here instead of metadata